### ISSUE TITLE
please answer my question

### ISSUE THREAD

**Issue by benlee73** [phamquiluan/publaynet#12]
I tried to train with main_training.py.

But I couldn't get the same performance weights as your trained weights in Benchmarking. Is it correct what you learned with the given main_training.py? What is the difference?

What is the number of GPUs and batch size used for training?
How much time did it take to train weights in Benchmarking?

Please be sure to answer my questions. Thank you very much.

**Comment by phamquiluan** [comment]
I don't remember. I used 4 V100 GPUs, it costs me about a week for each experiment.

If you want to beat me about the accuracy, you must have a powerful machine. Play with hyperparameters like image size, batch size, learning rate, optimizers,...

**Comment by benlee73** [comment]
Thank you so much for answering.
